### PR TITLE
Add Yieldmo bidder

### PR DESCRIPTION
**Pull request by RossGoncharuk**
Added Yieldmo bidder, configuration files and tests.

### CODE REVIEW

File: src/test/java/org/prebid/server/ApplicationTest.java [hunk 1]
```diff
@@ -492,6 +493,38 @@ public void openrtb2AuctionShouldRespondWithBidsFrom33Across() throws IOExceptio
...
+                .withRequestBody(equalToJson(jsonFrom("openrtb2/yieldmo/test-cache-yieldmo-request.json")))
+                .willReturn(aResponse().withBody(jsonFrom("openrtb2/yieldmo/test-cache-yieldmo-response.json"))));
+
+        // when
+        final Response response = given(spec)
+                .header("Referer", "http://www.example.com")
+                .header("X-Forwarded-For", "192.168.244.1")
+                .header("User-Agent", "userAgent")
+                .header("Origin", "http://www.example.com")
+                // this uids cookie value stands for {"uids":{"ym":"YM-UID"}}
```
Comment (rpanchyk) [inline src/test/java/org/prebid/server/ApplicationTest.java]: According to cookie family name `uid` should be "yieldmo".

File: src/main/java/org/prebid/server/bidder/yieldmo/proto/YieldmoImpExt.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@AllArgsConstructor(staticName = "of")
+@Value
+public class YieldmoImpExt {
+
+    @JsonProperty("placement_id")
```
Comment (rpanchyk) [inline src/main/java/org/prebid/server/bidder/yieldmo/proto/YieldmoImpExt.java]: No need for snake-case.

File: src/test/resources/org/prebid/server/ApplicationTest/openrtb2/yieldmo/test-auction-yieldmo-response.json [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+          "tid": "tid"
+        },
+        "regs": {
+          "ext": {
+            "gdpr": 0
+          }
+        },
+        "ext": {
+          "prebid": {
+            "aliases": {
```
Comment (rpanchyk) [inline src/test/resources/org/prebid/server/ApplicationTest/openrtb2/yieldmo/test-auction-yieldmo-response.json]: Aliases is not used in this test. Section can be removed.

File: src/main/java/org/prebid/server/bidder/yieldmo/YieldmoMetaInfo.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+ * Defines Yieldmo meta info
+ */
+public class YieldmoMetaInfo implements MetaInfo {
+
+    private BidderInfo bidderInfo;
+
+    public YieldmoMetaInfo(boolean enabled, boolean pbsEnforcesGdpr) {
+        bidderInfo = BidderInfo.create(enabled, "progsupport@yieldmo.com",
+                Collections.emptyList(), Collections.singletonList("banner"),
+                null, 0, pbsEnforcesGdpr);
```
Comment (rpanchyk) [inline src/main/java/org/prebid/server/bidder/yieldmo/YieldmoMetaInfo.java]: Pls set the GDPR vendor id from https://vendorlist.consensu.org/vendorlist.json

Comment (RossGoncharuk) [reply src/main/java/org/prebid/server/bidder/yieldmo/YieldmoMetaInfo.java]: Thanks for the link - I wasn't aware of it and just took what they've had in GO, which was 0.

**Pull request merged**